canbench-bin/src/main.rs

```diff
@@ -19,20 +19,28 @@ struct Args {
     // If true, only prints the benchmark results (and nothing else).
     #[clap(long)]
     less_verbose: bool,
+
+    // If provided, use the specified configuration file instead of the default one.
+    #[clap(long, value_parser = value_parser!(PathBuf))]
+    cfg_file_path: Option<PathBuf>,
```

Comment:
nit: for brevity, I'd shorten the name.

```suggestion
    cfg_path: Option<PathBuf>,
```

Reply:
Done.